content/3. Procedurele informatie/4. Techniek/Workshop Git/3. Git Forks & Branches.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 3. Git Forks & Branches
+taxonomie: None
+tags: 
+draft: true
+---
+## Opdracht 1: Forken
+>In de volgende opdrachten gaan we dieper in op de functies die Git(Hub) biedt. De eerste stap is het forken van een repository.
+
+**Stap 1:** Open de [**voorbeeld repository**](https://github.com/Windesheim-HBO-ICT/GIT-Workshop-Studenten).
```

Comment:
Voeg even een goede readme toe aan deze repo. Waar is de repo precies voor? Sommige docenten of studenten kunnen hem misschien zien en denk van ja wat moet ik hiermee? Desnoods ook een link naar deze repo / de website